mods/d2k/rules/defaults.yaml

```diff
@@ -88,68 +90,35 @@
 			Spice: 80
 			SpiceBlobs: 80
 			Dune: 70
-		ROT: 5
-	SelectionDecorations:
-	Selectable:
-		Bounds: 32,32
-	TargetableUnit:
-		TargetTypes: Ground, C4
-	Passenger:
-		CargoType: Vehicle
-	AttackMove:
-	HiddenUnderFog:
-	GivesExperience:
-	DrawLineToTarget:
-	ActorLostNotification:
 	ProximityCaptor:
```

Comment:
D2k doesn't need `ProximityCaptor` at all (at the off chance that someone does decide to make a CTF map, they can add the trait in there).
Please remove them all.